client/src/containers/LoggedPage.js

```diff
@@ -10,15 +11,16 @@ import EventsHistory from '../components/EventsHistory'
 import useLocalStorage from "use-local-storage";
 
 import NexmoClient from "nexmo-client";
+import * as _ from'lodash';
```

Comment:
we are importing the entire lodash only for deepclone. 

let's use something like this: 

https://www.npmjs.com/package/deep-clone